### ISSUE TITLE
ConfusionMatrix axis - labels

### ISSUE THREAD

**Issue by mfoglio** [pytorch/ignite#2195]
I am looking at the [ConfusionMatrix documentation](https://pytorch.org/ignite/generated/ignite.metrics.confusion_matrix.ConfusionMatrix.html ) and I can't understand what axis corresponds to the _predicted values_ and which one to the _actual values_.

In other words, which one of the two following options is correct?
1) https://miro.medium.com/max/1000/1*fxiTNIgOyvAombPJx5KGeA.png
2) https://miro.medium.com/max/1400/1*85t6zbUiQA0fotnhDaJLaA.png

**Comment by vfdev-5** [comment]
Thanks for the question, @mfoglio ! It should a standard definition: lines are predictions and rows are targets. The second image: https://miro.medium.com/max/1400/1*85t6zbUiQA0fotnhDaJLaA.png

```python
import torch
from ignite.metrics import ConfusionMatrix

cm = ConfusionMatrix(2)
# update receives (y_preds, y_true)
cm.update((torch.tensor([[0.0, 1.0], [0.0, 1.0]]), torch.tensor([0, 1])))
cm.update((torch.tensor([[0.0, 1.0], [0.0, 1.0]]), torch.tensor([0, 1])))
cm.compute()
> 
tensor([[0, 2],
             [0, 2]])
```

**Comment by mfoglio** [comment]
Thank you @vfdev-5 . I would love to update the doc. Should I create a new branch from `master` or simply commit my edit to `master`? This would be my first time contributing to open source project :) Thanks for the help.

**Comment by vfdev-5** [comment]
@mfoglio you can do the following:
- fork from this repository
- clone your fork locally and create a branch : `git clone <your-fork>`, `git checkout -b fix-docs-2195`
- try to render docs locally: https://github.com/pytorch/ignite/blob/master/CONTRIBUTING.md#writing-documentation
  - you can skip katex installation
- make doc changes
- rerender docs and check your modification
- once everything looks good, commit changes: `git commit -a` and add message: "Fixed #2195, improved ConfusionMatrix docs"
- push to your fork and create a PR with Github: `git push origin fix-docs-2195`

HTH

Some refs may worth to take a look:
- https://github.com/pytorch/ignite/blob/master/CONTRIBUTING.md#quickstart-guide-for-first-time-contributors
- https://github.com/pytorch/ignite/blob/master/CONTRIBUTING.md#writing-documentation
- https://github.com/firstcontributions/first-contributions

**Comment by mfoglio** [comment]
Thank you @vfdev-5 ! I am almost done. What upstream branch should I use? `fix-docs-2195`?

**Comment by vfdev-5** [comment]
@mfoglio sounds good! You can use any name, for example `fix-docs-2195`

**Comment by mfoglio** [comment]
Pull request is [here](https://github.com/pytorch/ignite/pull/2196). If I did anything wrong, let me know how can I do better next time.
